bodhi/server/templates/update.html

```diff
@@ -1033,7 +1033,13 @@ <h3>Related Bugs <span class="badge">${len(update.bugs)}</span></h3>
         % for build in update.builds:
         <tr class="media">
           <td>
-            <a href="https://koji.fedoraproject.org/koji/search?terms=${build.nvr}&type=build&match=glob" target="_blank">
+            <%
+              import urlparse
+
+              build_url = urlparse.urljoin(request.registry.settings.get('koji_web_url'),
+                                           'koji/search?terms='+build.nvr+'&type=build&match=glob')
```

Comment:
It looks like Fedora's Koji is configured to be served at ```https://koji.fedoraproject.org/koji/```, which would mean that the ```koji_web_url``` setting should be set to that value, which I think means that we should drop ```koji/``` from this join.